### PR TITLE
fix: workaround React native command not sent

### PR DESCRIPTION
**Pull request by krocard**
React Native's `UIManager.dispatchViewManagerCommand` sometimes (~3/4 times on emulator, very rarely on recent devices) doesn't sent native commands. They get stuck until the next UI refresh.

This issue should be fixed in the latest React Native release but our SDK needs significant work to update to this version.

Until the React Native version is updated, force UI redraws after dispatching commands to force them to be received by the native side.

Player issue: https://github.com/bitmovin/bitmovin-player-react-native/issues/163
RN issue: https://github.com/microsoft/react-native-windows/issues/7543

### CODE REVIEW

File: src/components/PlayerView/index.tsx [hunk 1]
```diff
@@ -57,6 +57,7 @@ function dispatch(command: string, node: NodeHandle, ...args: any[]) {
     Platform.OS === 'android'
       ? (UIManager as any).NativePlayerView.Commands[command].toString()
       : UIManager.getViewManagerConfig('NativePlayerView').Commands[command];
+  //console.log("dispatching command %s", command)
```
Comment (matamegger) [inline src/components/PlayerView/index.tsx:60]:
```suggestion
```

I guess we can remove this.

File: src/components/PlayerView/index.tsx [hunk 2]
```diff
@@ -76,6 +77,17 @@ export function PlayerView({
   ...props
 }: PlayerViewProps) {
   // Native view reference.
+  const fakeStateUpdater = useState(1)[1];
```
Comment (matamegger) [inline src/components/PlayerView/index.tsx:80]: The comment above is no longer at the right position.. that said, I think we can remove it anyways.

File: src/components/PlayerView/index.tsx [hunk 3]
```diff
@@ -76,6 +77,17 @@ export function PlayerView({
   ...props
 }: PlayerViewProps) {
   // Native view reference.
```
Comment (krocard) [inline src/components/PlayerView/index.tsx:79]:
```suggestion
```

File: src/components/PlayerView/index.tsx (hunk 2 above)
Comment (krocard) [reply src/components/PlayerView/index.tsx:78]: Done: d87a4452dc7eaccf426b4d696fb3f28889b19b29

**Review by matamegger — APPROVED** [review]

**Review by rolandkakonyi — APPROVED** [review]
I just jumped in uninvited to test this on iOS :) 
It still works, so this is good to go from my side with the comment by Matthias addressed.

**Pull request merged**